### ISSUE TITLE
`0.11.2` missing `amd64` image version

### ISSUE THREAD

**Issue by skos-ninja** [jtblin/kube2iam#366]
It looks like the `0.11.2` version of kube2iam is missing the `amd64` image version and only has published the `arm64` version and as such machines using the `latest` tag would fail to pull the image due to the missing image

**Comment by jetersen** [comment]
@jtblin maybe you don't have notification configured so excuse me for poking you :)

The above is still the case 😓

![image](https://github.com/jtblin/kube2iam/assets/1661688/a8c9d867-95be-42f8-9880-e3f6db41342e)

**Comment by ryshoooo** [comment]
@jtblin It's actually a bit messier than just amd64 missing. For the latest tag, arm64/v8 points to the same image as 0.11.2, while arm64 points to 0.11.1. So even the latest is not quite the latest for arm64.

**Comment by jtblin** [comment]
I think I messed up with the manifest and it's creating issues. The `amd64` image shows as `arn64` on docker hub, even with pushing new ones so not sure what's happening. Travis CI hasn't been working for the repo anymore since they changed to travis-ci.com so I'm in the process of moving to CircleCI, hopefully that will help fixing the manifests and pushing images for each architecture.

**Comment by YoninL** [comment]
We are still using the kube2iam-2.6.0 version but seems the tag is now gone. Will try to use 0.11.1 first.

Failed to pull image "jtblin/kube2iam:kube2iam-2.6.0": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/jtblin/kube2iam:kube2iam-2.6.0": failed to resolve reference "docker.io/jtblin/kube2iam:kube2iam-2.6.0": docker.io/jtblin/kube2iam:kube2iam-2.6.0: not found

**Comment by jtblin** [comment]
My bad, did some cleanup and deleted that one.

**Comment by jtblin** [comment]
Fixed the issues with `0.11.2`, it nows has both `arm64` and `amd64`, I had to move the build to CircleCI and use `docker buildx` as I couldn't get travis-ci to work for this open source project. I am going to discontinue the `kube2iam-amd64` and `kube2iam-arm64` as they are redundant and not used much. The arm images are available in the main `kube2iam` version. I updated `latest` to point to `0.11.2` which should be pretty stable (it was only an `alpine` base image update).

Still trying to get the workflows working for tags so I haven't been able to release a `dev` tagged image but will do that later manually.
